app/components/CheckoutForm.js

```diff
@@ -10,6 +10,7 @@ function CheckoutForm({ stripe }) {
     addingToCart,
     cartAmount,
     cartCurrency,
+    cartDispatch,
```

Comment:
@ynnoj perhaps this could be exported as `resetCart` instead. The Context in my opinion should take care of invoking `dispatch` and return functions that dispatch internally.